flask_peewee/rest/__init__.py

```diff
@@ -92,7 +96,7 @@ class RestResource(object):
     expose_registry = False
 
     # http methods supported for `edit` operations.
-    edit_methods = ('PATCH', 'PUT', 'POST')
+    edit_methods = ('PUT', 'POST')
```

Comment:
Please ignore this change. It was accidentally introduced in the postgres tests PR that this is based on.